client/src/models/opportunityfilterproxymodel.cpp

```diff
@@ -183,5 +183,21 @@ bool OpportunityFilterProxyModel::lessThan(const QModelIndex &left, const QModel
             return leftDt < rightDt;
         }
     }
-    return FilterProxyModel::lessThan(left, right);
+
+    if (FilterProxyModel::lessThan(left, right)) {
+        return true;
+    } else {
+        QVariant l = (left.model() ? left.model()->data(left, sortRole()) : QVariant());
+        QVariant r = (right.model() ? right.model()->data(right, sortRole()) : QVariant());
+
+        // if itens are equal sort by creation date
```

Comment:
typo: itens -> items

But this isn't really about items, it's about values (in the sort column) ;)